### PR TITLE
Added prettier

### PR DESCRIPTION
**Pull request by SanjeevLakhwani**

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -1,15 +1,30 @@
...
+
+      - name: Install dependencies
         run: npm install
-      - name: Run linter
+
+      - name: Run EsLint
         run: npm run lint
+
+      - name: Run Prettier
+        run: npm run format -- --check
```
Comment (davidlougheed) [inline .github/workflows/lint.yml:30]: missing trailing newline

**Review by davidlougheed — CHANGES_REQUESTED** [review]
fix all issues caught

File: src/Components/Charts/BentoPie.tsx [hunk 2]
```diff
@@ -49,13 +51,16 @@ const BentoPie = ({
...
-  chartThreshold = useChartThreshold(),
-  maxLabelChars = useChartMaxLabelChars(),
+  chartThreshold,
+  maxLabelChars,
   ...params
 }: PieChartProps) => {
   const t = useChartTranslation();
   const { fill: theme } = useChartTheme().pie[colorTheme];
 
+  const resolvedChartThreshold = useResolvedValue(chartThreshold, useChartThreshold);
```
Comment (davidlougheed) [inline src/Components/Charts/BentoPie.tsx:61]: i don't think passing a hook as an argument is recommended.

**Review by davidlougheed — CHANGES_REQUESTED** [review]

File: src/Components/Charts/BentoPie.tsx (hunk 2 above)
Comment (SanjeevLakhwani) [reply src/Components/Charts/BentoPie.tsx]: I guess I over engineered it

**Review by davidlougheed — APPROVED** [review]
lgtm

**Pull request merged**